packages/@sanity/desk-tool/src/actions/PublishAction.tsx

```diff
@@ -152,6 +189,15 @@ export const PublishAction: DocumentActionComponent = (props) => {
       ? null
       : title,
     shortcut: disabled || publishScheduled ? null : 'Ctrl+Alt+P',
-    onHandle: handle,
+    onHandle: documentIsReferenced ? () => setShowConfirmPublishReferencedDocument(true) : handle,
+    dialog: showConfirmPublishReferencedDocument ? dialog : undefined,
   }
 }
+
+const LinkBox = styled.div`
+  border-color: #e8f1fe;
```

Comment:
I think this should use a color from the theme values instead. Same with the grey below for the border 👍